core/src/main/java/io/questdb/std/Hash.java

```diff
@@ -57,6 +57,12 @@ public static int hashLong128(long key1, long key2) {
         return (int) (h ^ h >>> 32);
     }
 
+    public static int hashLong256(long key1, long key2, long key3, long key4) {
+        long h = key1 * M2 + key2 + key3 + key4;
```

Comment:
This hash function doesn't look correct. It should be an unrolled version of `hashMem32` when it's called on 32 bytes chunk of memory.